crates/wdk-build/src/utils.rs

```diff
@@ -26,6 +26,30 @@ use windows::{
 
 use crate::{ConfigError, CpuArchitecture};
 
+/// Type for versions of the format "major.minor"
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub struct BasicVersion {
```

Comment:
Consider renaming it to `TwoPartVersion`. The word "Basic"  in `BasicVersion` does not convey any meaning.